provider_shopper/lib/screens/cart.dart

```diff
@@ -85,11 +85,24 @@ class _CartTotal extends StatelessWidget {
             SizedBox(width: 24),
             TextButton(
               onPressed: () {
-                ScaffoldMessenger.of(context).showSnackBar(
-                    SnackBar(content: Text('Buying not supported yet.')));
+                  ScaffoldMessenger.of(context).showSnackBar(SnackBar(
```

Comment:
This code doesn't look like it's formatted correctly. Can you run `flutter format` from the command line or trigger a reformat in your IDE?

Reply:
this was the code I deleted to  resolve the merge request directly from the Github website that is why it is not formatted I guess